oarepo_runtime/cli/configuration.py

```diff
@@ -7,14 +7,21 @@
 
 from .base import oarepo
 
+def create_config_dict(config):
+    """Recursively create configuration dict without LocalProxies inside."""
+    result = {}
+    for k, v in config.items():
+        if isinstance(v, dict):
+            result[k] = create_config_dict(v)
+        elif not isinstance(v, LocalProxy):
```

Comment:
Can we have a list here as well? I'd rename the method to remove_lazy_objects and let it accept any kind of argument - dict, list, ... - and inside decide if it is a dict, list or something else and handle it accordingly. The lazy might be an item in a list as well (though that is probably not the case in the current configuration).